pages/vi/nextcloud-tor.md

```diff
@@ -12,7 +12,7 @@
 
 ## Prepare Treehouses image
 
-Download the latest treehouses image from http://download.treehouses.io, then use [balenaEtcher](https://etcher.io) to flash the image onto your SD card.
+Download the latest treehouses image [here](./../download.md), then use [balenaEtcher](https://etcher.io) to flash the image onto your SD card.
```

Comment:
`pages/download.md` is what we want

also @law-lin it would be awesome if you allowed us to edit your PR ;)

Reply:
@dogi I see.
 How do I allow you to edit my PR? I looked it up and I don't see the "Allow edits from maintainers" option on this page.

Reply:
Would be on your own repository page -> settings i think